docker/Dockerfile-test-scripts

```diff
@@ -1,5 +1,5 @@
-# test via
-#   docker build -t install_scripts:test -f docker/Dockerfile-test-scripts .
+# test e.g. via
+#   docker build --pull -t mirai/smarp_install_scripts:test -f docker/Dockerfile-test-scripts .
```

Comment:
This is actually meant only for local testing w/o any connection to a repo, hence the simple `-t install_scripts:test`.

Reply:
ok, makes sense